etl/steps/data/garden/literacy/2025-06-11/historic_literacy_omm.meta.yml

```
@@ -0,0 +1,42 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    unit: "%"
+    short_unit: "%"
+    presentation:
+      topic_tags:
+        - Global Education
+    display:
+      numDecimalPlaces: 1
+    description_processing: |-
```

I would add a 'processing_level: major' - currently this shows up as 'minor' in the chart. 

```suggestion
    processing_level: major
    description_processing: |-
```